app/controllers/concerns/part_of_claim_journey.rb

```diff
@@ -25,23 +25,41 @@ def send_unstarted_claimants_to_the_start
     redirect_to current_policy.start_page_url unless current_claim.persisted?
   end
 
+  # Keep this for StudentLoans and MathsAndPhysics journeys
+  # Keep ECP journey working for now with current_claim as we build out combined journey
   def current_claim
     @current_claim ||= claim_from_session || build_new_claim
   end
 
+  # Placeholder needs session work
+  def current_ecp_claim
+    @current_ecp_claim
+  end
+
+  # Placeholder needs session work
+  def current_lup_claim
+    @current_lup_claim
+  end
```

Comment:
Suggestion: something more generic like `current_claims` being either an array, or hash of `Policy` => `Claim` objects

Reply:
This would make sense when doing the work which addresses storing the claim_ids in the session and adapting the  `claims_controller` to deal with multiple claims